### PR TITLE
Update README.md - remove repo specification from README.md's Gemfile

### PR DESCRIPTION
**Pull request by take**
According to https://rubygems.org/gems/query_reviewer/versions/0.1.8, there's no need to specify the repo in Gemfile.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# QueryReviewer #
+    # QueryReviewer #
```
Comment (take) [inline README.md:1]: ooops i have no idea why this diff slipped in, gonna remove it

**Comment by nesquena** [comment]
It's OK I think I got the gist of what you were trying to get changed with the commit above. I include both options since sometimes I will pull in fixes but not release a new version.

**Comment by nesquena** [comment]
Oh and thanks!

**Comment by take** [comment]
:smile:

**Pull request closed without merging**